### PR TITLE
Option to read new description from stdin

### PR DESCRIPTION
**Pull request by untitaker**
Will add tests if you approve of API

### CODE REVIEW

**Comment by untitaker** [comment]
@WhyNotHugo any opinion on this? I use this feature everyday

File: todoman/cli.py [hunk 1]
```diff
@@ -318,11 +318,18 @@ def invoke_command(click_ctx, command):
     callback=_validate_list_param,
     help='The list to create the task in.'
 )
+@click.option(
+    '--read-description',
+    '-r',
+    is_flag=True,
+    default=False,
+    help='Read task description from stdin'
```
Comment (WhyNotHugo) [inline todoman/cli.py]: Please finish sentences with `.` in help texts.

File: tests/helpers.py [hunk 2]
```diff
@@ -1,9 +1,9 @@
+import pytest
```
Comment (WhyNotHugo) [inline tests/helpers.py]: The was actually right before:

https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names

File: tests/test_cli.py [hunk 3]
```diff
@@ -873,3 +873,12 @@ def test_edit_raw(todo_factory, runner):
 
     assert not result.exception
     assert not result.output
+
+
+def test_new_description_from_stdin(runner):
+    result = runner.invoke(cli, ['new', '-l', 'default', '-r', 'hello'],
+                           input='world\n')
+    assert not result.exception
```
Comment (WhyNotHugo) [inline tests/test_cli.py]: I believe it's also important to check that the word `world` ends up in the right field in the created todo. Probably can be done following the pattern [here](https://github.com/pimutils/todoman/blob/05b7f915be7b04fb16a0c1fb5940d5f504cc7add/tests/test_cli.py#L709-L710).

**Comment by WhyNotHugo** [comment]
Don't forget about the CHANGELOG entries 😉

File: tests/helpers.py (hunk 2 above)
Comment (untitaker) [reply tests/helpers.py]: I fixed it because Flake8 was complaining.

**Comment by untitaker** [comment]
I think everything is fixed now. The stylecheck will fail because the import order is "wrong" according to flake8 though.

File: tests/test_cli.py [hunk 4]
```diff
@@ -873,3 +873,12 @@ def test_edit_raw(todo_factory, runner):
 
     assert not result.exception
     assert not result.output
+
+
+def test_new_description_from_stdin(runner, todos):
+    result = runner.invoke(cli, ['new', '-l', 'default', '-r', 'hello'],
+                           input='world\n')
+    assert not result.exception
+    todo, = todos()
```
Comment (WhyNotHugo) [inline tests/test_cli.py]: Nice, hand't thought of using `todo,`, a lot cleaner than my approach! 😉

**Pull request merged**